flink-kubernetes-operator/pom.xml

```diff
@@ -47,6 +47,20 @@ under the License.
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-client</artifactId>
             <version>${fabric8.version}</version>
+            <!-- FLINK-29384, can be removed when fabric8 pulls snakeyaml 1.32+ -->
+            <exclusions>
+                <exclusion>
+                    <groupId>org.yaml</groupId>
+                    <artifactId>snakeyaml</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <!-- FLINK-29384, can be removed when fabric8 pulls snakeyaml 1.32+ -->
+        <dependency>
```

Comment:
Could you please share me why we include the dependency explicitly and not use the `dependencyManagement`?

Reply:
I tried the dependency management route in `flink-kubernetes-parent`, it did not seems to work. I see your point, and can give it a try if you prefer. Given that it is hopefully just a temporary fix I would suggest to leave it as is.